### ISSUE TITLE
Biting people is silly loud and gives away vampires since they only use it

### ISSUE THREAD

**Issue by CptWad** [vgstation-coders/vgstation13#20326]
if were gonna keep the noise have it disabled for vampires since you cant sneak a sucky otherwise

**Comment by Pathid** [comment]
Alternatively the range of the sound could be reduced.

**Comment by DamianX** [comment]
I thought it was the same sound used by biting, though.

**Comment by Pathid** [comment]
Vampire bites and other bites are handled by two different ``playsound()`` calls.

**Comment by DamianX** [comment]
Ok but they're the same `playsound(loc, 'sound/weapons/bite.ogg', 50, 1, -1)`
So it's incorrect to say only vampires use it.

**Comment by Pathid** [comment]
He meant it doesn't see practical use outside of vamps, something like that.

**Comment by strawberrytan** [comment]
after your first suck you have heat vision and you can be far more aware of whether someone will hear the succ or not

**Comment by CptWad** [comment]
yeah vampires are really the only people who use bite intent (besides a few exceptions when biting is robust) so if someone is say being operated on or passes out in the bar they IMMEDIATELY know that someone bit them and that means vampire
I watched some poor doctor get ousted by sec after he took a nibble out of a detective in surgery
its also a matter of validhunting but it can also be solved by making biting far quieter or make vampires silent biters.

**Comment by Pathid** [comment]
I mean, they shouldn't be made aware of it at all if they're unconscious. 
Kinda leads to the larger issue if having that little circle of vision and sound available is even worth it if people are gonna use it to metagame all the time.

**Comment by robintime** [comment]
duplicate of #10089, but with better discussion
